src/diagram/HostedWebEditor.tsx

```diff
@@ -25,6 +25,7 @@ interface HostedWebEditorProps {
   projectName: string;
   embedded?: boolean;
   baseURL?: string;
+  readOnlyMode?: boolean;
```

Comment:
should this _also_ make it so the editing tools don't pop up, or equation isn't editable?

Reply:
It's a design choice we need to make! InsightMaker makes the model _truly_ read only (no edits whatsoever). I think it's a mistake, because you then need an account to fork if you want to change even the tiniest thing.

I believe it's a fantastic design choice to let the "visitor" mess with the model as much as they want, given that nothing will be saved, and that they are not interacting with the server in any way. This also gives a "soft feature" of anonymous/"account-less" modelling (which you might not want ahah).

What do you think?

Reply:
Agreed! I'll give it a check

Reply:
Done! I think it looks pretty good for now :)